src/module.c

```diff
@@ -384,6 +384,44 @@ int TSDB_createRule(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     return REDISMODULE_OK;
 }
 
+/*
+TS.CORRELATE key_1 key_2
+*/
+int TSDB_correlate(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
Add `RedisModule_AutoMemory(ctx);` to the start of the function, this will release redis created resources in the end of the command run,